apps/appgen/template_Make.defs

```diff
@@ -0,0 +1,21 @@
+###########################################################################
+#
+# Copyright 2018 Samsung Electronics All Rights Reserved.
```

Comment:
This is an awesome tool for TizenRT newbies.
But, please double check whether the year '2018' is reasonable in case that some app developers will create app in 2019.